starwave/starwave.py

```diff
@@ -189,12 +187,18 @@ def get_cmd(self, nstars, gr_dict, pdict):
 
         output_mags = input_mags + self.asdf_noise[idxs]
 
-        nans = np.isnan(output_mags).any(axis = 1)
+        output_bad = np.isnan(output_mags).any(axis = 1)
 
```

Comment:
output_bad is not necessary, define an output_good instead so you don't have to negate it twice immediately below

Reply:
on the 2 places where you added a `~` in one case you added a parenthesis, in the other not. I am not sure whether the order of operations matters (`.any` vs` ~`). Did you verify that you get the same histograms as before in the plot?